Description: fix compilation failure of this header in arm (where VFP is enabled)
Author: James Clarke <jrtc27@jrtc27.com>
Forwarded: https://github.com/opencv/opencv/pull/8976
Applied-Upstream: https://github.com/opencv/opencv/commit/25020f267236ab8e715fabbbdea5b81e39a01120

--- a/modules/core/include/opencv2/core/fast_math.hpp
+++ b/modules/core/include/opencv2/core/fast_math.hpp
@@ -72,7 +72,7 @@
         int res; \
         float temp; \
         (void)temp; \
-        asm(_asm_string : [res] "=r" (res), [temp] "=w" (temp) : [value] "w" (_value)); \
+        __asm__(_asm_string : [res] "=r" (res), [temp] "=w" (temp) : [value] "w" (_value)); \
         return res
     // 2. version for double
     #ifdef __clang__
